doc/extension_store/manifest.jsonc

```diff
@@ -0,0 +1,145 @@
+{
+    // Info displayed in the Extension Store
+    "id": "postgres-vs",
+    "name": "PostgreSQL Virtual Schema",
+    "description": "...",
+    "version": "2.0.1",
+    // Dependencies
+    "bucketFsUpload": {
+        "jdbcDriver": {
+            // Download URL for the JDBC driver, e.g. Maven Central URL
+            "downloadUrl": "https://jdbc.postgresql.org/download/postgresql-42.3.3.jar",
+            // License URL
+            "licenseUrl": "https://github.com/pgjdbc/pgjdbc/blob/master/LICENSE",
+            // True if the user must agree to the license before installation, e.g. for Oracle JDBC driver
+            "licenseAgreementRequired": false,
+            // Path and filename when uploading files to BucketFS
+            "bucketFsPath": "/buckets/${BFS_SERVICE}/${BUCKET}/virtual-schema/postgresql-jdbc-driver.jar"
+        },
+        "virtualSchemaAdapter": {
+            "downloadUrl": "https://github.com/exasol/postgresql-virtual-schema/releases/download/2.0.1/virtual-schema-dist-9.0.4-postgresql-2.0.1.jar",
+            "licenseUrl": "https://github.com/exasol/postgresql-virtual-schema/blob/main/LICENSE",
+            "bucketFsPath": "/buckets/${BFS_SERVICE}/${BUCKET}/virtual-schema/postgresql-adapter.jar"
+        }
+    },
+    "parameters": {
+        "adapterSchema": {
+            "type": "string", // normal text field
+            "description": "Schema in which to install the virtual schema adapter",
+            "default": "POSTGRES_ADAPTER_SCHEMA",
+            "scope": "adapter",
+            "readOnlyAfterInstallation": true // only editable during installation
+        },
+        "adapterScriptName": {
+            "type": "string",
+            "description": "Name of the adapter script",
+            "default": "POSTGRES_ADAPTER",
+            "scope": "adapter",
+            "readOnlyAfterInstallation": true
+        },
+        "connectionName": {
+            "type": "string",
+            "description": "Name of the PostgreSQL connection containing the PostgreSQL credentials",
+            "default": "POSTGRESQL_CONNECTION",
+            "scope": "connection",
+            "readOnlyAfterInstallation": true
+        },
+        // We could think about specifying host, port and database separetly
+        "jdbcUrl": {
+            "type": "string",
```

Comment:
This is a good example of a situation where we should either plan for a dedicated type or provide a validation regex.
Regex is more flexible, type allows the UI better diagnostics in case of mismatch.